### PR TITLE
1318 admin quickstart

### PR DESCRIPTION
**Pull request by elliothershberg**
This adds the admin quickstart guide to our docs (#1318 )

Covers:

- Launching `admin-server`
- Adding an assembly using assembly manager GUI
- Accessing configuration editor for a track, seeing live updates

Open to any suggestions on improvements :)

### CODE REVIEW

**Comment by elliothershberg** [comment]
Restarted build after dotplot test failed

**Comment by cmdcolin** [comment]
- can you optimize your images with pngquant
- can you remove the part about editing the "height" variable on the sequence track? this is too detailed I think. on some level the briefer the content, the better
- is there any reason why the assembly dialog box has such a large area?

**Comment by elliothershberg** [comment]
> * can you optimize your images with pngquant

Yeah, I can do that now
> * can you remove the part about editing the "height" variable on the sequence track? this is too detailed I think. on some level the briefer the content, the better

Sure, so just stick with showing how to access the config settings but not specific example?

> * is there any reason why the assembly dialog box has such a large area?

For parity with the button visually

**Comment by cmdcolin** [comment]
>Sure, so just stick with showing how to access the config settings but not specific example?

Ya, or I might suggest focus on the assembly manager rather than switch to configuration editor at all maybe?



>>is there any reason why the assembly dialog box has such a large area?
>For parity with the button visually

Not sure I was specific, I was looking at the dialog box header which is very large I mean
https://github.com/GMOD/jbrowse-components/pull/1397/files?short_path=ad824d1#diff-ad824d17eafecf14f93d5119c25248c6ff47d4ebb9576c6204c2851e416ae5bf

That is not new on this branch but I wonder can it be made smaller

File: website/docs/quickstart_admin.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+### Pre-requisites
+
+This tutorial requires having the following software installed
+
+- [JBrowse 2 CLI tools](quickstart_web#install-the-cli-tools)
+
+- [JBrowse 2 web application](quickstart_web#using-jbrowse-create-to-install-jbrowse)
+
+### Starting JBrowse2 Admin Server
```
Comment (cmdcolin) [inline website/docs/quickstart_admin.md:17]: this is fairly picky but can we make subheadings use sentence-casing, e.g.  ```### Starting JBrowse2 admin server```

Comment (elliothershberg) [reply website/docs/quickstart_admin.md:17]: Sure, it should be consistent with other guides

**Comment by elliothershberg** [comment]
Yeah I could work on making the dialog box header smaller at some point, probably because the back arrow isn't on same line as the header

**Comment by cmdcolin** [comment]
I pushed a change to make the header bar smaller. Would you be able to remake the screenshots?

**Comment by elliothershberg** [comment]
Yeah sure

**Review by garrettjstevens — APPROVED** [review]

**Comment by elliothershberg** [comment]
Merge?

**Pull request merged**